animate.py

```diff
@@ -56,7 +56,33 @@ def mst(G):
 ax_graph  = fig.add_subplot(gs[0])
 ax_plot = fig.add_subplot(gs[1])
 values = []
-
+def entropy():
```

Comment:
let's do it differently here. No need to iterate over all dates twice. First create entropy function that calculates entropy for the given graph. Then in line 113 use this function instead of maximum edge witdth